iphone/Maps/UI/PlacePage/Components/PlacePagePreviewViewController.swift

```diff
@@ -165,7 +165,7 @@ final class PlacePagePreviewViewController: UIViewController {
         details = nil;
       }
       
-      setScheduleLabel(state: L("editor_time_open"),
+      setScheduleLabel(state: L(""),
```

Comment:
```suggestion
      setScheduleLabel(state: "",
```

Even better is to rewrite setScheduleLabel and remove the first argument.

Reply:
<img width="410" alt="Screenshot 2023-01-10 at 2 46 56 PM" src="https://user-images.githubusercontent.com/71179696/211678444-17d3e3aa-88ee-4b3f-9070-0ae2639af8a9.png">
<img width="416" alt="Screenshot 2023-01-10 at 2 46 44 PM" src="https://user-images.githubusercontent.com/71179696/211678453-c7cd397f-0574-4670-97d6-c01581c4f5e0.png">

Reply:
> Even better is to rewrite setScheduleLabel and remove the first argument.
for this case I can't remove the first argument in setScheduleLabel

  case .allDay:
      setScheduleLabel(state: L("twentyfour_seven"),
                       stateColor: UIColor.systemGreen,
                       details: nil);